### PR TITLE
feat: update walkthrough

### PR DESCRIPTION
**Pull request by anchenyi**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/30383720

image
image

### CODE REVIEW

File: packages/vscode-extension/src/handlers/copilotChatHandlers.ts [hunk 1]
```diff
@@ -241,11 +241,36 @@ export async function invokeTeamsAgent(args?: any[]): Promise<Result<null, FxErr
...
-    triggerFromProperty["trigger-from"] === TelemetryTriggerFrom.CommandPalette
-      ? "@teamsapp Use this GitHub Copilot extension to ask questions about Teams app and agent development."
-      : "@teamsapp Write your own query message to find relevant templates or samples to build your Teams app and agent as per your description. E.g. @teamsapp create an AI assistant bot that can complete common tasks.";
+  let query = "";
+  switch (triggerFromProperty[TelemetryProperty.TriggerFrom]) {
+    case TelemetryTriggerFrom.TreeView:
+      query =
+        "@teamsapp Use this GitHub Copilot extension to ask questions about Teams app and agent development.";
+      break;
+    case TelemetryTriggerFrom.CommandPalette:
```
Comment (yuqizhou77) [inline packages/vscode-extension/src/handlers/copilotChatHandlers.ts:247]: can these two conditions be put together?

Comment (anchenyi) [reply packages/vscode-extension/src/handlers/copilotChatHandlers.ts:247]: Yes, I will put them together. Thanks!

**Review by yuqizhou77 — DISMISSED** [review]

**Review by jayzhang — DISMISSED** [review]

**Review by tecton — DISMISSED** [review]

File: packages/vscode-extension/media/walkthroughs/get-started/create/step2.png
Comment (supkasar) [inline packages/vscode-extension/media/walkthroughs/get-started/create/step2.png:1]: Updated content for 'Bot' is - Create instant, engaging chat experiences that automate tasks seamlessly.

File: packages/vscode-extension/media/walkthroughs/get-started/create/step2_with_chat.png
Comment (supkasar) [inline packages/vscode-extension/media/walkthroughs/get-started/create/step2_with_chat.png:1]: Same here

File: packages/vscode-extension/media/walkthroughs/get-started/create/step2.png
Comment (anchenyi) [reply packages/vscode-extension/media/walkthroughs/get-started/create/step2.png:1]: @supkasar, thanks for the suggestions. I have updated the content.

**Review by supkasar — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Pull request merged**